whylog_vim/exceptions.py

```diff
@@ -0,0 +1,12 @@
+class WhylogVimExceptions(Exception):
+    pass
+
+
+class UnknownAction(WhylogVimExceptions):
+
+    def __init__(self, action_type, state):
+        self.action_type = action_type
+        self.state = state
+
+    def __repr__(self):
+        return 'Unknown action for argument name %s in states %s' % (self.action_type, self.state)
```

Comment:
No, no, no xD! Please rename 'argument name' with proper argument name. Probably should be: 'argument_name' -> 'action_type'